kong/clustering/rpc/concentrator.lua

```diff
@@ -90,6 +93,67 @@ local function enqueue_notifications(notifications, notifications_queue)
 end
 
 
+function _M:process_one_response(payload)
+  assert(payload.jsonrpc == jsonrpc.VERSION)
+  local payload_id = payload.id
+
+  -- response
+  local cb = self.interest[payload_id]
+  self.interest[payload_id] = nil -- edge trigger only once
+
+  if not cb then
+    ngx_log(ngx_WARN, "[rpc] no interest for concentrator response id: ", payload_id, ", dropping it")
+    return
+  end
+
+  local res, err = cb(payload)
+  if not res then
+    ngx_log(ngx_WARN, "[rpc] concentrator response interest handler failed: id: ",
+            payload_id, ", err: ", err)
+  end
+end
+
+
+function _M:process_one_request(target_id, reply_to, payload, collection)
+  local payload_id = payload.id
```

Comment:
We need to handle invalid input in this case, say if the payload is not of type table
Please refer to the spec: https://www.jsonrpc.org/specification#overview:~:text=rpc%20call%20with%20an%20invalid%20Batch%20(but%20not%20empty)%3A

Reply:
done, thanks.